nifi-assembly/NOTICE

```diff
@@ -1543,6 +1543,7 @@ The following binary components are provided under the MIT License.  See project
 
   (MIT License) EWS Java API (com.microsoft.ews-java-api:ews-java-api:2.0 - https://github.com/OfficeDev/ews-java-api)
   (MIT License) libffi (libffi-3.2.1 - http://sourceware.org/libffi/)
+  (MIT License) InfluxDB (org.influxdb:influxdb-java:2.7 https://github.com/influxdata/influxdb-java/)  
```

Comment:
there should not be any (MIT License) section in our NOTICE.  These all belong in the LICENSE.  I will move these other ones in the current master though